app/directives/forms/edit/edit-side-event.html

```diff
@@ -854,6 +854,7 @@ <h2>Responsible Person </h2>
                       <span ng-show="(doc.meta.status=== 'published' || doc.meta.status=== 'scheduled')">Edit Your Live Side-event</span>
                     </button>
                 </div>
+<pre>{{doc|json}}</pre>
```

Comment:
remove debug

Reply:
@blaisefonseca resolved.  thanks for the catch